templates/claim_payment.html

```diff
@@ -36,6 +42,19 @@ <h6>You can claim a payment of {{payment.amount|int2asset}} {{ config["ASSET_NAM
                                 </div>
                                 <button type="submit" class="btn btn-primary">Submit</button>
                             </form>
+			    <br/>
```

Comment:
fix up inconsistant line spacing in this block

Reply:
should be fixed